src/resolve-config-editorconfig.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const editorconfig = require("editorconfig");
+const mem = require("mem");
+
+const editorconfigAsyncNoCache = (filePath, config) =>
+  Promise.resolve(
+    filePath &&
+      !config &&
+      editorconfig.parse(filePath).then(editorConfigToPrettier)
```

Comment:
https://www.npmjs.com/package/editorconfig#parsefilepath-options

We might need to set `root` when the path is on a UNC share.

Reply:
Nice catch, fixed in https://github.com/prettier/prettier/pull/2760/commits/8776fa668a1353c61461c46331d7eddfd9545741